agents/dev/run_smtp.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import print_function
+
+import argparse
+import json
+
+from agents.smtp.smtp import SMTPAgent
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--host',
+        default='127.0.0.1',
+        help='SMTP server host'
+    )
+    parser.add_argument(
+        '--port',
+        type=int,
+        default=25,
+        help='SMTP port'
+    )
+    parser.add_argument(
+        '--controller-url',
+        dest='controller_url',
+        help='Controller URL'
+    )
+    parser.add_argument(
+        '--processes',
+        help='Comma-separated list of processes'
+    )
```

Comment:
Support `--processes` being either comma-separated or multiple flags:
```
parser.add_argument(
    '--processes',
    nargs='*',
    help='List of processes to check (e.g., --processes postfix exim sendmail)'
)
```
Then you don’t need to split manually.